vitest.config.ts

```diff
@@ -6,6 +6,6 @@ export default defineConfig({
   test: {
     include: ["./**/*.test.ts"],
     globals: true,
-    environment: "jsdom",
+    environment: "node",
   },
```

Comment:
Ah. This is where I wonder if we’ll run into problems not having different vitest configs per sub-project

Reply:
As of now its all fine I've checked `caravan-client` and its running correctly with same configurations 
![image](https://github.com/user-attachments/assets/5e4f229b-ba54-4838-b934-22b98e0182f8)

Reply:
But can cause a problem in `caravan-wallet` package since there in some places `window` object is used which is not supported in node environment

Reply:
Got a solution for this : )
https://vitest.dev/guide/environment#environments-for-specific-files

Reply:
I've found a fix for this from vitest docs, we can define environment for specific files as well  :)
https://vitest.dev/guide/environment#environments-for-specific-files

Reply:
Oh nice. Maybe we can add this in some documentation somewhere so we don't lose track of it for the future? Maybe a "testing" section in the main readme?

Reply:
sounds cool, let me edit the readme file : )